os_brick/initiator/connectors/lightos.py

```diff
@@ -64,20 +66,39 @@ def __init__(self,
 
     @staticmethod
     def get_ip_addresses():
-        ip_addresses = []
-        result = subprocess.check_output(['ip', 'addr']).decode('utf-8')
-        lines = result.split('\n')
-        for line in lines:
-          if 'inet ' in line:
-              parts = line.split()
-              ip_parts = parts[1]
-              ip_parts = ip_parts.split('/')
-              host_ip=ip_parts[0]
-              if '127.0.0.1' in host_ip:
-                 continue
-              ip_addresses.append(host_ip)
-
-        return ip_addresses
+        """
+        Find all IPs for the host machine, return list of IP addresses.
+        """
+        loop_back_ips=['127.0.0.1','0:0:0:0:0:0:0:1', '::1']
+        ips = []
+        is_ipv6_enabled = netutils.is_ipv6_enabled()
+                          
+        ifaces = interfaces()
+        for iface in ifaces:
+            addresses = ifaddresses(iface)
+            
+            for address_family in (AF_INET, AF_INET6):
+                family_addresses = addresses.get(address_family)
```

Comment:
`addresses.get(address_family, None)` unless you are absolutely sure that `addresses` always has the `address_family`.